apollo_checker/report_local.sh

```diff
@@ -12,10 +12,13 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Automatic config
 tmp_dir=`mktemp -d`
 raw_apollo_gff="/data/raw_apollo.gff"
+organism_name="Genus Species"
 cur_date=`date +%F_%X`
 
 cd "$tmp_dir"
 
+echo -n "" > "$output_dir/full_report.json"
```

Comment:
and this?

Reply:
Create the file if not existing, empty it if it exist (without newline for the regex)